### PR TITLE
Update CLI import option documentation

### PR DESCRIPTION
**Pull request by dominikl**
See also https://github.com/ome/omero-documentation/issues/2190

### CODE REVIEW

File: omero/downloads/cli/help.out [hunk 1]
```diff
@@ -1,49 +1,141 @@
...
+  $ omero import foo.tiff -- --debug=ALL
+  # Display used files for importing foo.tiff
+  $ omero import foo.tiff -f
+  # Limit debugging output
+  $ omero import -- --debug=ERROR foo.tiff
 
 For additional information, see:
 https://docs.openmicroscopy.org/latest/omero/users/cli/import.html
-Report bugs at https://www.openmicroscopy.org/forums
+Report bugs to <ome-users@lists.openmicroscopy.org.uk>
```
Comment (jburel) [inline omero/downloads/cli/help.out]: I don't think that what should be recommended

Comment (dominikl) [reply omero/downloads/cli/help.out]: That's what `--help` prints out.

Comment (jburel) [reply omero/downloads/cli/help.out]: In that case we should update the output of the command

Comment (jburel) [reply omero/downloads/cli/help.out]: see https://github.com/ome/omero-py/pull/297

**Comment by jburel** [comment]
@dominikl I pushed a commit to this branch if you want to have a look

**Comment by dominikl** [comment]
👍 Looks good!

**Pull request merged**